.github/workflows/ci.yaml

```
@@ -1,108 +1,219 @@
+          go-version: "1.24.x"
+          check-latest: true
+
+      - name: Check Go Version
+        run: go version
+
+      - name: Ensure go.sum
+        working-directory: ${{ env.SERVICE_PATH }}
+        run: go mod tidy
+
+      - name: Cache Go Modules
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)